core/src/operations/mod.rs

```diff
@@ -372,7 +374,7 @@ impl Operation {
             Self::FmpAdd    => 0b0000_0110,
             Self::MLoad     => 0b0000_0111,
             Self::Swap      => 0b0000_1000,
-            // <empty>      => 0b0000_1001,
+            Self::TapeUpd   => 0b0000_1001,
```

Comment:
We don't need to create a new instruction for this. Setting an active tape should be done via a decorator, and thus, does not need a "real" operation.